packages/devtools/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "devtools",
+  "version": "5.7.5",
```

Comment:
will the version automatically be updated on release to 5.12 (or higher)?

Reply:
yes, once I release it in a bundle with the other projects